src/Microsoft.Extensions.Configuration.Binder/ConfigurationBinder.cs

```diff
@@ -60,6 +60,15 @@ public static object Get(this IConfiguration configuration, Type type)
         /// Attempts to bind the given object instance to configuration values by matching property names against configuration keys recursively.
         /// </summary>
         /// <param name="configuration">The configuration instance to bind.</param>
+        /// <param name="key">The configuration key to bind.</param>
```

Comment:
Yeah I was going to name the parameter section, until I saw the other parameter names